### ISSUE TITLE
Clarification regarding classifier protocol

### ISSUE THREAD

**Issue by prachigarg23** [arthurdouillard/dytox#24]
Hi @arthurdouillard , thanks for your work! In the paper and supplementary, I could not completely understand the classifier protocol. 

I found the usage of sigmoid and BCE for training task-specific classifier heads really interesting. In section 3.3, you mention late fusion but I wanted to confirm what happens at test time (given this is a class-incremental learning setting and task-ID is unknown at inference). In the code, there seems to be a separate individual classifier as well as joint expanding classifier with #classes expanding. 

Do you simply concatenate the logits from all task-specific classifiers? Any clarification will be great.

**Comment by arthurdouillard** [comment]
Yep, i just concat logits and argmax over it.
